src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -117,6 +120,41 @@ const ParcelsPage: React.FC<{}> = () => {
         })();
     }, []);
 
+    function getValueBoundaries<DataType>(values: (DataType | null)[]): number[] {
+        const valueSet = new Set(values);
+
+        const boundaries: number[] = [];
+        valueSet.forEach((value) => {
+            if (value === values[0]) {
+                return;
+            }
+
+            const indexOfFirstOccurrence = values.indexOf(value);
+            if (indexOfFirstOccurrence === -1) {
+                //
```

Comment:
what is the purpose of this comment? can we restructure this logic so that we only have an if, and not an if/else with an empty if?

Reply:
was going to include a `throw` if the value cannot be found but that basically cannot happen so I removed it